docs/source/federated-types/federated-directives.mdx

```diff
@@ -102,13 +133,31 @@ The default value is `true`.
 </tbody>
 </table>
 
+### `@extends`
+
+```graphql
+directive @extends on OBJECT | INTERFACE
+```
+
+Indicates that an object or interface definition is an extension of _another_ definition of that same type.
+
+> ⚠️ **If you define your subgraph schema in a static text file, do not use this directive!** Instead, use GraphQL's built-in `extend` keyword.
```

Comment:
unsure if this is true, I think the general rule is -> use `@extends` only if your subgraph library does not support the `extend` keyword syntax

Reply:
Cool, tweaked the language here